src/utils/random-user.tsx

```diff
@@ -0,0 +1,11 @@
+const baseUrl = 'https://randomuser.me/api/?seed=motivation&inc=picture,name';
+
+export const getUsers = async (count: number = 20, page: number = 1) => {
+  try {
+    const users = await fetch(`${baseUrl}&page=${page}&results=${count}`);
+    const parsedUsers = await users.json();
+    return parsedUsers.results;
+  } catch (error) {
+    alert(error);
```

Comment:
Use Alert from react-native. Also, this function is about getting users, so IMO it would make sense to return empty array in case if request fails.